util/util.go

```diff
@@ -236,7 +236,7 @@ func DetachAnyLoopbackDevice(file string) error {
 }
 
 func ValidateName(name string) bool {
-	validName := regexp.MustCompile(`^[a-zA-Z0-9][a-zA-Z0-9_.-]+$`)
+	validName := regexp.MustCompile(`^[a-zA-Z0-9]([a-zA-Z0-9_.-]|@)+$`)
 	return validName.MatchString(name)
```

Comment:
Maintainability: Don't return raw values because it makes more work when debugging later.

Suggested change:

    isValid := validName.MatchString(name)
    return isValid